app/controllers/accounts_controller.rb

```diff
@@ -30,13 +29,10 @@ def destroy
 
   private
 
-  def account_params
-    params.require(:account)
-          .permit(:subdomain,
-                  owner_attributes: [
-                    :first_name, :last_name, :email,
-                    :password, :password_confirmation]
-                 )
+  def signup_params
+    params.require(:signup)
+          .permit(:subdomain, :first_name, :last_name, :email,
+                    :password, :password_confirmation)
```

Comment:
Since we're also whitelisting this in the form object, I think we could pass the entire `params` hash to it safely.